util/io/files/tar.go

```diff
@@ -158,10 +159,10 @@ func (t *tgz) tgzFile(path string, fi os.FileInfo, err error) error {
 		return fmt.Errorf("relative path error: %s", err)
 	}
 
-	if t.inclusions != nil && base != "." && !fi.IsDir() {
+	if t.inclusions != nil && len(t.inclusions) > 0 && base != "." && !fi.IsDir() {
 		included := false
 		for _, inclusionPattern := range t.inclusions {
-			found, err := filepath.Match(inclusionPattern, base)
+			found, err := doublestar.Match(inclusionPattern, relativePath)
```

Comment:
I'm not sure we can safely switch glob interpreters.

Reply:
The inclusions were only used on CLI for the `app diff` command. I've replaced the default values to match the glob pattern.